learn/tanssi/technical.md

```diff
@@ -1,53 +1,95 @@
\ No newline at end of file
+As mentioned, the Tanssi network and the ContainerChains are sibling chains with no hierarchical dependency. They are communicated only via the relay chain, and therefore, the relay chain plays a key role in the protocol.
+
+By accessing the data stored in the finalized blocks of the relay chain and cross-referencing headers, the collators can learn their assignation for the session, the ContainerChains can confirm that a certain group of collators from Tanssi has been assigned to them, and Tanssi can verify that the author of a container block was the expected one and reward accordingly.
+
+The Tanssi protocol relays on the relay chain as a means to provide the necessary data to both, Tanssi and its ContainerChains, allowing them to collaborate and validate the correctness of the block production service.
+
+![Relay chain](/images/learn/tanssi/technical/technical-3.png)
+
+### The Role of the Tanssi Network {: #tanssi-newtwork } 
+
+As previously discussed, the Tanssi protocol assigns collators to the Tanssi network itself and the ContainerChains, and the result of this assignment is stored within the chain state.
+
+Another important piece of information that Tanssi stores is the latest header for every ContainerChain. This data is read from the relay chain and, being stored in every Tanssi block, it allows the protocol to keep track of the state in every chain and also to identify and reward accordingly the collator that produced their last block.
+
+![Tanssi Network](/images/learn/tanssi/technical/technical-4.png)
+
+### The Role of the ContainerChain {: #containerchain } 
+
+As a collator node assigned to a ContainerChain deployed in Tanssi has built-in Tanssi node functionality, it is technically feasible to read the state from the Tanssi network and the blocks from the relay chain.
+
+Leveraging this ability to access the states, the current collator with the authority to produce a block will read the latest block produced in the relay chain, which contains the state root of the latest block produced in Tanssi. With this state root, it will proceed to read the state in Tanssi, and include in the block of the ContainerChain the latest state root of the Tanssi network, the current set of collators assigned to the ContainerChain, and its public signature, allowing Tanssi to know who produced the block and reward the collator.
+
+Once the block is completed with the ContainerChain transactions, it will be proposed as a candidate and handed over to the relay chain validators, which will ensure that the included state proofs match the state proofs from the latest state of Tanssi (preventing unauthorized collation) and that the transactions produced valid state transitions. Having verified the work of the collator, the relay chain will finalize the proposed block, including its candidate receipt in the relay chain block.
+
+![ContainerChain collation](/images/learn/tanssi/technical/technical-5.png)
```

Comment:
Stay consistent with the design of the image. This one you used a white background for the boxes